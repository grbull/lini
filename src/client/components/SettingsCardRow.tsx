import cn from 'classnames';
import React, { ReactElement, ReactNode } from 'react';

interface Props {
  children: ReactNode;
}

export function SettingsCardRow({ children }: Props): ReactElement {
  return (
    <div className={cn('flex', 'justify-between', 'items-center', 'py-1')}>
      {children}
    </div>
  );
}
