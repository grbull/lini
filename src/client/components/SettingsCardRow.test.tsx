/**
 * @jest-environment jsdom
 */

import '@testing-library/jest-dom/extend-expect';

import { render, RenderResult } from '@testing-library/react';
import React from 'react';

import { SettingsCardRow } from './SettingsCardRow';

function setup(): RenderResult {
  const utils = render(<SettingsCardRow>More testing</SettingsCardRow>);

  return { ...utils };
}

describe('SettingsCardRow Component', () => {
  it('matches the snapshot', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
