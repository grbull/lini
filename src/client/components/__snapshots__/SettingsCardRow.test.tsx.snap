// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsCardRow Component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex justify-between items-center py-1"
  >
    More testing
  </div>
</DocumentFragment>
`;
